toolbox/fdc3-conformance/Metadata-Tests.md

```diff
@@ -0,0 +1,62 @@
+| Test   | 6.Confirm2    | An `instanceId` should be provided, confirm that it matches the one in `appIdentifier2`  |
+
+- `AppInstanceMetadata`: Perform the above steps
+
+## Finding Instances
+
+| App | Step           | Details                                                                                           |
+|-----|----------------|---------------------------------------------------------------------------------------------------|
+| Test   | 1.Open1    | Open the first instance of App A using <br/> `const appIdentifier1 = await fdc3.open({appId: "<A's appId>"})` <br/>and retrieve its `AppIdentifier` with instance details.  |
+| Test   | 2.Open2    |Open a second instanceo of App A using <br>`const appIdentifier2 = await fdc3.open({appId: "<A's appId>"})` <br/> and retrieve its `AppIdentifier` with instance details.  Confirm that the `instanceId` differs from the first instance.  |
+| Test   | 3.FindInstances    | Retrieve details of open instances with <br/> `let instances = await fdc3.findInstances({appId: "<A's appId>"})` <br/> confirm that both `appIdentifier1` and `appIdentifier2` are both present in the array.  |
+| Test   | 4.RaiseIntent   | Use `appIdentifier1` to raise an intent and target that instance, with<br/> `const resolution = fdc3.raiseIntent("aTestingIntent", {"type": "testContextX"}, appIdentifier1)` |
+| Test   | 5.Confirm1 | Check that `resolution.source` matches `appIdentifier1` |
+| A | 6.ConfirmReceipt | Ensure that the instance of app A represented by `appIdentifier1` received the raised intent |
+
+- `FindInstances`: Perform the above steps
+
+## Getting Info For The Agent
+
+| App | Step           | Details                                                                                           |
+|-----|----------------|---------------------------------------------------------------------------------------------------|
+| Test   | 1.getInfo    |Retrieve the `ImplementationMetadata` for the DesktopAgent with <br/> `fdc3.getInfo().then((implMetadata) => {  ... subsequent steps ...}`<br>**Note that the use of `then` is deliberate and intended to confirm that a promise returned (as this function switched from synchronous to asynchronous in 2.0)**|
+| Test   | 2.CheckVersion  | Check that the `fdc3Version` variable is present and at or greater than 2.0 (which you can do with the [`versionIsAtLeast` function from FDC3's Methods.ts](https://github.com/finos/FDC3/blob/add64f8302c6dcdc8437cf0e245101e927b69ec2/src/api/Methods.ts#L207):<br>`const isFDC3v2 = versionIsAtLeast(implMetadata, "2.0")`  |
+| Test   | 3.CheckProvider  | Check that the `provider` variable is present and not an empty string  |
+| Test   | 4.CheckFeatures  | Check that the `optionalFeatures`, `optionalFeatures.OriginatingAppMetadata` and `optionalFeatures.UserChannelMembershipAPIs` variables are all present and that the latter two provide boolean values  |
+
+- `GetInfo1`: Perform the above steps
```

Comment:
Test applies to both versions (apart from step 4)
```suggestion
- `GetInfo1`: ![1.2](https://img.shields.io/badge/FDC3-1.2-green) ![2.0](https://img.shields.io/badge/FDC3-2.0-blue) Perform the above steps
```

Reply:
See previous comment - while I doubt this is going to cause anyone any trouble, it does change the 1.2 test pack.

Reply:
As above it probably should go in, was a small oversight